jhsingle_native_proxy/main.py

```diff
@@ -172,6 +173,11 @@ def get_port_from_env():
 def run(port, destport, ip, presentation_path, debug, logs, authtype, request_timeout, last_activity_interval, force_alive, ready_check_path, 
         ready_timeout, repo, repobranch, repofolder, conda_env, allow_root, notebookapp_allow_origin, forward_user_info, query_user_info, progressive, 
         websocket_max_message_size, command):
+    
+    # enable_pretty_logging sets StreamHandler and Formatter for the specified loggers so that Jhsingle-native-proxy logs are formatted, sent to sys.stderr, and then outputted in the console. 
+    print("Setting pretty logging")
+    tornado.log.enable_pretty_logging(logger=app_log)
```

Comment:
Since you have imported `enable_pretty_logging` already, you should just use
```
enable_pretty_logging(logger=app_log)
```

Reply:
Made this change in the new commit